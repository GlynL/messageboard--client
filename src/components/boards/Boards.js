import React from "react";
import { Link } from "react-router-dom";
import BoardList from "./BoardList";

export default () => {
  return (
    <section>
      <header>
        <h1>All Boards</h1>
        <p>Click on a board to view threads.</p>
        <Link to="/new-board">Start a new Board.</Link>
      </header>
      <BoardList />
    </section>
  );
};
