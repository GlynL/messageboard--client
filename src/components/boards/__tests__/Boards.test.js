import React from "react";
import { shallow } from "enzyme";
import { Boards } from "../Boards";

it("matches snapshot w/out auth", () => {
  const wrapper = shallow(<Boards authed="" />);
  expect(wrapper).toMatchSnapshot();
});
it("matches snapshot w/ auth", () => {
  const wrapper = shallow(<Boards authed="mockauth" />);
  expect(wrapper).toMatchSnapshot();
});
