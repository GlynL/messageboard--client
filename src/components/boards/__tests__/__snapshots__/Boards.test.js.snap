// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot w/ auth 1`] = `
<section
  className="boards"
>
  <header
    className="boards__header"
  >
    <h1
      className="boards__header-title"
    >
      All Boards
    </h1>
    <p>
      Click on a board to view threads.
    </p>
    <Link
      replace={false}
      to="/new-board"
    >
      Start a new Board.
    </Link>
  </header>
  <Connect(BoardList) />
</section>
`;

exports[`matches snapshot w/out auth 1`] = `
<section
  className="boards"
>
  <header
    className="boards__header"
  >
    <h1
      className="boards__header-title"
    >
      All Boards
    </h1>
    <p>
      Click on a board to view threads.
    </p>
    <p>
      Sign up to start a new board.
    </p>
  </header>
  <Connect(BoardList) />
</section>
`;
